libraries/rtos/rtx/TARGET_CORTEX_A/RTX_CM_lib.h

```diff
@@ -388,14 +528,13 @@ __attribute__((naked)) void software_init_hook (void) {
 
 extern int  __low_level_init(void);
 extern void __iar_data_init3(void);
+extern void __iar_dynamic_initialization(void);
+extern void mbed_sdk_init(void);
 extern void exit(int arg);
 
-__noreturn __stackless void __cmain(void) {
+void mbed_main(void) {
```

Comment:
OK I found mbed_main declaration in retarget. But what about these iar startup functions, why they are declared above if I dont see them used anywhere, same for mbed_sdk_init ? mbed_main does only kernel init, thread start and kernel start.